src/assets/pages/blogposts/post-48.html

```diff
@@ -0,0 +1,31 @@
+<img src="https://querido-diario-static.nyc3.cdn.digitaloceanspaces.com/post0048/post-48-grafico-1.png">
+<p><strong>Aprendizados</strong></p>
+<p><span style="font-weight: 400;">Devido ao foco em novos reposit&oacute;rios para recebimento de contribui&ccedil;&otilde;es, alguns aprendizados aconteceram na pr&aacute;tica e devem ser levados em considera&ccedil;&atilde;o na Hacktoberfest de 2024.&nbsp;</span></p>
+<p><strong>Frontend.&nbsp;</strong><span style="font-weight: 400;">Lucas Guimar&atilde;es, mantenedor do reposit&oacute;rio querido-diario-frontend, destacou que nesse reposit&oacute;rio ser&aacute; importante ficar atento n&atilde;o apenas para a quantidade de contribui&ccedil;&otilde;es, mas tamb&eacute;m para a qualidade do que est&aacute; sendo desenvolvido:</span></p>
+<p><em><span style="font-weight: 400;">&ldquo;Houve novas adi&ccedil;&otilde;es ao produto vindas de sugest&otilde;es de novas funcionalidades feitas por colaboradores rec&eacute;m-chegados ao projeto, sem que necessariamente as sugest&otilde;es propostas estivessem alinhadas com a gest&atilde;o e fizessem sentido como um todo. Utilizar melhor as labels para especificar as caracter&iacute;sticas da issue e tentar estabelecer um mecanismo de validar as entradas com os mantenedores, aceitando-as ou n&atilde;o&rdquo;</span></em><span style="font-weight: 400;">, destacou.</span></p>
+<p><span style="font-weight: 400;">Para lidar com essa quest&atilde;o, Pedro Guimar&atilde;es refor&ccedil;a que </span><em><span style="font-weight: 400;">&ldquo;no ano que vem devemos ter uma melhor gest&atilde;o pr&eacute;via do reposit&oacute;rio, com prepara&ccedil;&atilde;o e cria&ccedil;&atilde;o de issues e maior dedica&ccedil;&atilde;o na aplica&ccedil;&atilde;o da label &ldquo;good first issue&rdquo;, que indica as issues mais f&aacute;ceis&rdquo;, </span></em><span style="font-weight: 400;">explica.</span></p>
+<p><strong>Feedbacks da comunidade</strong></p>
+<p><span style="font-weight: 400;">No geral, os feedbacks foram bastante positivos para a participa&ccedil;&atilde;o do Querido Di&aacute;rio na Hacktoberfest deste ano, em especial com rela&ccedil;&atilde;o ao engajamento das pessoas nos eventos promovidos e movimenta&ccedil;&atilde;o no servidor. Para Juliana Trevine, a forma&ccedil;&atilde;o de grupos de trabalho voltados para &aacute;reas espec&iacute;ficas do projeto pode ter ajudado. Tr&ecirc;s </span><a href="https://queridodiario.ok.org.br/blog/post/41"><span style="font-weight: 400;">GTs iniciaram atividades</span></a><span style="font-weight: 400;"> em agosto deste ano e seguem at&eacute; o final deste m&ecirc;s. A expectativa &eacute; que essa din&acirc;mica continue fazendo parte da comunidade daqui para frente.&nbsp;</span></p>
```

Comment:
`&nbsp;` no final da linha como esse é espaço em branco que pode ser removido
```suggestion
<p><span style="font-weight: 400;">No geral, os feedbacks foram bastante positivos para a participa&ccedil;&atilde;o do Querido Di&aacute;rio na Hacktoberfest deste ano, em especial com rela&ccedil;&atilde;o ao engajamento das pessoas nos eventos promovidos e movimenta&ccedil;&atilde;o no servidor. Para Juliana Trevine, a forma&ccedil;&atilde;o de grupos de trabalho voltados para &aacute;reas espec&iacute;ficas do projeto pode ter ajudado. Tr&ecirc;s </span><a href="https://queridodiario.ok.org.br/blog/post/41"><span style="font-weight: 400;">GTs iniciaram atividades</span></a><span style="font-weight: 400;"> em agosto deste ano e seguem at&eacute; o final deste m&ecirc;s. A expectativa &eacute; que essa din&acirc;mica continue fazendo parte da comunidade daqui para frente.</span></p>
```